packer/ansible/roles/cyhy_feeds/tasks/main.yml

```diff
@@ -0,0 +1,87 @@
+---
+# tasks file for cyhy_feeds
+- name: Install unzip
+  apt:
```

Comment:
Prefer `package` to `apt` when you're not using any apt-specific options.  This makes the file more likely to work with non-Debian systems.